targets/targets.json

```diff
@@ -8937,6 +8937,32 @@
             "network-default-interface-type": "WIFI"
         }
     },
+    "CY8CKIT_064S2_4343W": {
+        "inherits": ["MCU_PSOC6_M4"],
+        "features": ["BLE"],
+        "components_remove": ["QSPIF"],
```

Comment:
Please confirm - is there a Quad SPI Flash on this board?  If there is, it would be very important to support it.  This is extremely useful for FOTA.

Reply:
Quad SPI flash is available on this board and will be supported. It is disabled right now because of current BSP implementation - it interferes with Secure Boot implementation. Once this issue will be negotiated and fixed - QSPI will be enabled for this target with separate PR.

Reply:
okayh